package.json

```diff
@@ -37,47 +37,47 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@babel/core": "7.20.12",
-    "@babel/eslint-parser": "7.19.1",
+    "@babel/core": "7.22.5",
+    "@babel/eslint-parser": "7.22.5",
     "@chbphone55/classnames": "2.0.0",
     "@eik/cli": "^2.0.22",
-    "@semantic-release/changelog": "6.0.2",
+    "@semantic-release/changelog": "6.0.3",
     "@semantic-release/git": "10.0.1",
     "@warp-ds/uno": "^1.0.0-alpha.45",
-    "@warp-ds/component-classes": "1.0.0-alpha.106",
-    "autoprefixer": "10.4.13",
+    "@warp-ds/component-classes": "1.0.0-alpha.110",
+    "autoprefixer": "10.4.14",
     "cors": "2.8.5",
-    "cssnano": "5.1.14",
+    "cssnano": "6.0.1",
```

Comment:
Major upgrade here, have we done some smoke tests or something? Are we good with it?

Reply:
Actually, this one is not used anywhere in the project. I removed it 🙌